frameworks/svelte/__tests__/testUtils/testUtils.ts

```diff
@@ -3,20 +3,60 @@ import {render, RenderResult} from '@testing-library/svelte'
 import {testWithMockedIntersectionObserver} from '../../../../testUtils/setupIntersectionObserverMock';
 import type {SvelteComponentDev} from "svelte/internal";
 
+/**
+ * Get element from container
+ * @param container
+ * @param type
+ */
 const getElement = (container: HTMLElement, type: string): HTMLElement => container.querySelector(type) || {} as HTMLElement;
 
+/**
+ * Get img element from container
+ * @param container
+ */
 const getImageElement = (container: HTMLElement): HTMLImageElement => getElement(container, 'img') as HTMLImageElement;
 
-const getImageAttr = (container: HTMLElement, attr: string): any => ((getImageElement(container) || {}) as any)[attr];
-
+/**
+ * Get element attribute from container
+ * @param container
+ * @param type
+ * @param attr
+ */
 const getElementAttr = (container: HTMLElement, type: string, attr: string): any => ((getElement(container, type) || {}) as any)[attr];
 
+/**
+ * Get img attribute from container
+ * @param container
+ * @param attr
+ */
+const getImageAttr = (container: HTMLElement, attr: string): any => getElementAttr(container, 'img', attr);
+
+
+/**
+ * Mount given component on client side
+ * @param cmp
+ * @param props
+ * @return render result
+ */
 const mount = async (cmp?: typeof SvelteComponentDev, props?: any): Promise<RenderResult> => {
   const result = cmp ? render(cmp, props) : {};
   await tick();
   return result as RenderResult;
 };
 
+/**
+ * Mount given component on server side
+ * @param cmp
+ * @param props
+ * @return render result
+ */
+const mountSSR = (cmp: any, props: any = {}): string => {
```

Comment:
for cmp, is there a better type for `any` ? 
for props, if we must be very generic, can we use `Record<string, any>` to represent the object?